mobx_codegen/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: mobx_codegen
 description: Code generator for MobX that adds support for annotating your code with @observable, @computed, @action and also creating Store classes.
-version: 1.0.1
+version: 1.0.2
```

Comment:
We are almost done here. Just need to add the entry in the changelog for `mobx_codegen`

Reply:
I added myself in the contributors list :-)